docs/architecture/grpc-for-wcf-developers/why-grpc.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Why gRPC is recommended for WCF developers
+description: gRPC for WCF Developers | Why gRPC is recommended for WCF developers
+author: markrendle
+ms.date: 09/02/2019
+---
+
+# Why gRPC is recommended for WCF developers
+
+Before diving deeply into the language and techniques of gRPC it is worth pulling together some ideas about why, given there are alternatives available, gRPC is the right solution for WCF developers who want to migrate to .NetCore.
+
+[Mark to provide text about similarities]
+
+## Benefits of gRPC
```

Comment:
+ Interoprability

There are gRPC libraries for C++, Java, Go, Node, Python, etc, etc. Proto files can be reused anywhere.